docs/source/guide/tutorials/base/how_to_train/instance_segmentation.rst

```diff
@@ -1,2 +1,327 @@
 Instance Segmentation model
-================================
\ No newline at end of file
+================================
+
+This tutorial reveals end-to-end solution from installation to model export and optimization for instance segmentation task on a specific example.
+On this page we show how to train, validate, export and optimize Mask-RCNN model on toy dataset.
+
+.. note::
+
+  To learn how to deploy the trained model, refer to :doc:`../deploy`.
+
+  To learn how to run the demo and visualize results, refer to :doc:`../demo`.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- CUDA Toolkit 11.4
+
+*************************
+Setup virtual environment
+*************************
+
+You can follow the installation process from a :doc:`quick_start guide <../../../get_started/quick_start_guide/installation>` to create a virtual environment.
+
+***************************
+Dataset preparation
+***************************
+
+1. Let's use the simple toy dataset `Car, Tree, Bug dataset <https://github.com/openvinotoolkit/training_extensions/tree/develop/tests/assets/car_tree_bug>`_ 
+provided by OTX.
+
+This dataset contains images of simple car, tree, bug with the annotation for instance segmentation.
+
+- ``car``	- Car Shape Illustration
+- ``tree``	- Tree Shape Illustration
+- ``bug``	- Bug Shape Illustration
+
+This allows us to look at the structure of the dataset used in instance-segmentation, and can be a good starting point for how to start an instance segmentation task with OTX.
+
+|
+
+.. image:: ../../../../../utils/images/car_tree_bug_gt_sample.png
+  :width: 400
+
+|
+
+2. Check the file structure of downloaded repository,
```

Comment:
what is the "downloaded repository"? Please, correct that

Reply:
Yes I will fix this.